pkgs/top-level/all-packages.nix

```diff
@@ -28884,6 +28884,8 @@ with pkgs;
 
   rtw88-firmware = callPackage ../os-specific/linux/firmware/rtw88-firmware { };
 
+  r8152-udev-rules = callPackage ../os-specific/linux/r8152-udev-rules { };
+
```

Comment:
instead of doing this you can also just add the package to `pkgs/by-name` but it's fine either ways just wanted to let you know since it's more comfortable to use the by-name thing.